frontend/src/scenes/surveys/constants.ts

```diff
@@ -11,6 +11,12 @@ export const SurveyQuestionLabel = {
     [SurveyQuestionType.MultipleChoice]: 'Multiple choice select',
 }
 
+export const SurveyUrlMatchTypeLabels = {
+    [SurveyUrlMatchType.Contains]: 'contains',
+    [SurveyUrlMatchType.Regex]: 'matches regex',
+    [SurveyUrlMatchType.Exact]: 'matches exactly',
```

Comment:
Can we make `matches exactly` -> `equals` instead? So it's just like our regular posthog filters! If you want to spruce things up even more and add the symbols too, that would be a great addition.

<img width="215" alt="Screen Shot 2023-10-03 at 11 39 00 AM" src="https://github.com/PostHog/posthog/assets/25164963/970843b2-cb0d-4b34-9c37-7d5728db051f">